.github/workflows/formatting.yaml

```diff
@@ -3,15 +3,24 @@ name: OpenTurbine-CI
 on: push
 
 jobs:
-  ClangFormat:
+  clang-format:
     runs-on: ubuntu-latest
+
     steps:
-    - name: Clone
+    - name: Checkout repository
       uses: actions/checkout@v4
-    - name: Check formatting
-      uses: DoozyX/clang-format-lint-action@v0.16.2
-      with:
-        source: './src ./tests/unit_tests'
-        exclude: '.'
-        extensions: 'cpp,hpp'
-        clangFormatVersion: 16
+
+    - name: Run clang-format
+      run: |
+        find . -regex '.*\.\(cpp\|hpp\|c\|h\)' -exec clang-format -i {} \;
```

Comment:
Would the following be a little cleaner?

```
run: |
        FILES=$(find . -name '*.cpp' -o -name '*.hpp' -o -name '*.c' -o -name '*.h')
        clang-format --dry-run --Werror $FILES
```

Reply:
Possibly.  I'd have to try it out to make sure it behaves the same way.  I'll put it on my to-do list to try something like this out.